### PR TITLE
Add more docs and templates

### PR DESCRIPTION
**Pull request by xtreamwayz**
- [x] Move code of conduct
- [x] Move contributing
- [x] Add support
- [x] Add issue template
- [x] Add pull request template
- [x] Remove docs after installation

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
         "issues": "https://github.com/zendframework/zend-expressive-skeleton/issues",
         "source": "https://github.com/zendframework/zend-expressive-skeleton",
         "rss": "https://github.com/zendframework/zend-expressive-skeleton/releases.atom",
-        "slack": "https://zendframework-slack.herokuapp.com",
+        "chat": "https://zendframework-slack.herokuapp.com",
```
Comment (froschdesign) [inline composer.json]: Is `slack` or `chat` allowed?

See: https://github.com/composer/composer/blob/4d8b9be5b6a0bda0a3d24c0a6ff89d7ccb26f715/res/composer-schema.json#L461

Comment (xtreamwayz) [reply composer.json]: Good point and I'm not sure. We can always call it irc, because it's just irc 2.0 :)

Comment (weierophinney) [reply composer.json]: Schema pretty clearly only shows "irc". I'll make that change on merge.

**Comment by weierophinney** [comment]
Thanks, @xtreamwayz!

**Comment by weierophinney** [comment]
Cherry-picked into master.

**Pull request closed without merging**